api/V1/recordingUtil.js

```diff
@@ -187,8 +187,72 @@ async function addTag(request, response) {
   });
 }
 
+// reprocessAll expects request.body.recordings to be a list of recording_ids
+// will mark each recording to be reprocessed
+async function reprocessAll(request, response) {
+  recordings =  request.body.recordings
+  var responseMessage = {
```

Comment:
it's a good habit to use `const` to avoid unintended reassignment (`const` doesn't mean what it might in other languages)